### ISSUE TITLE
Improve docs to better explain how to procedurally set values

### ISSUE THREAD

**Issue by colinmegill** [chrvadala/react-svg-pan-zoom#7]
Let's assume that we don't want to use the zoom tool, but we wanted to click a button and have it zoom in. Obviously this _can_ be done because we control value and pass it to viewer. But how should it be done?

**Comment by chrvadala** [comment]
Hi,

With the version 1.* you have to use ViewerHelper, there are a lot of available methods and with this you have also a method to perform zoom. 

```
import {ViewerHelper} from 'react-svg-pan-zoom';
...
let zoom = 1.1;
let viewerX = viewerWidth / 2;
let viewerY = viewerHeight / 2;
let nextValue = ViewerHelper.zoom(value, zoom, viewerX, viewerY);
```

At this moment I'm working on react-svg-pan-zoom 2.0.0 and I'm improving remote controls (I know that now it's not well explained).

**Comment by colinmegill** [comment]
Perfect! Thanks so much!

**Comment by chrvadala** [comment]
You're welcome

**Comment by chrvadala** [comment]
released with version 2.0
